pkg/opensearch/client/client.go

```diff
@@ -126,9 +127,11 @@ var NewClient = func(ctx context.Context, ds *backend.DataSourceInstanceSettings
 
 	version, err := extractVersion(jsonData.Get("version"))
 	if err != nil {
-		return nil, fmt.Errorf("opensearch version is required, err=%v", err)
+		return nil, fmt.Errorf("version is required, err=%v", err)
```

Comment:
When request fails with error it read "OpenSearch error..." which could be confusing is someone has configured Elastic? For example when testing datasource it shows below even when Elastic is selected:

<img width="384" alt="Screen Shot 2021-10-05 at 08 53 40" src="https://user-images.githubusercontent.com/745532/135974745-b4956115-ddcb-4421-870d-c0f856cd0af7.png">

Reply:
That's true, also `undefined. undefined` is not really the best error message ever.
I'll create an issue to improve error handling in general.